projects/uitdatabank/reference/search.json

```diff
@@ -0,0 +1,1674 @@
+          },
+          {
+            "$ref": "#/components/parameters/minAge"
+          },
+          {
+            "$ref": "#/components/parameters/allAges"
+          },
+          {
+            "$ref": "#/components/parameters/audienceType"
+          },
+          {
+            "$ref": "#/components/parameters/availableFrom"
+          },
+          {
+            "$ref": "#/components/parameters/availableTo"
+          },
+          {
+            "$ref": "#/components/parameters/attendanceMode"
+          },
+          {
+            "$ref": "#/components/parameters/bookingAvailability"
+          },
+          {
+            "$ref": "#/components/parameters/calendarType"
+          },
+          {
+            "$ref": "#/components/parameters/createdFrom"
+          },
+          {
+            "$ref": "#/components/parameters/createdTo"
+          },
+          {
+            "$ref": "#/components/parameters/modifiedFrom"
+          },
+          {
+            "$ref": "#/components/parameters/modifiedTo"
+          },
+          {
+            "$ref": "#/components/parameters/creator"
+          },
+          {
+            "$ref": "#/components/parameters/dateFrom"
+          },
+          {
+            "$ref": "#/components/parameters/dateTo"
+          },
+          {
+            "$ref": "#/components/parameters/localTimeFrom"
+          },
+          {
+            "$ref": "#/components/parameters/localTimeTo"
+          },
+          {
+            "$ref": "#/components/parameters/embedCalendarSummaries"
+          },
+          {
+            "$ref": "#/components/parameters/facets"
+          },
+          {
+            "$ref": "#/components/parameters/regions"
+          },
+          {
+            "$ref": "#/components/parameters/coordinates"
+          },
+          {
+            "$ref": "#/components/parameters/distance"
+          },
+          {
+            "$ref": "#/components/parameters/bounds"
+          },
+          {
+            "$ref": "#/components/parameters/id"
+          },
+          {
+            "$ref": "#/components/parameters/locationId"
+          },
+          {
+            "$ref": "#/components/parameters/organizerId"
+          },
+          {
+            "$ref": "#/components/parameters/labels"
+          },
+          {
+            "$ref": "#/components/parameters/mainLanguage"
+          },
+          {
+            "$ref": "#/components/parameters/languages"
+          },
+          {
+            "$ref": "#/components/parameters/completedLanguages"
+          },
+          {
+            "$ref": "#/components/parameters/hasMediaObjects"
+          },
+          {
+            "$ref": "#/components/parameters/price"
+          },
+          {
+            "$ref": "#/components/parameters/minPrice"
+          },
+          {
+            "$ref": "#/components/parameters/maxPrice"
+          },
+          {
+            "$ref": "#/components/parameters/sortScore"
```

Comment:
I think it will be difficult to group the sort parameters as another heading like `Query Parameters` but for example `Sort Parameters`?

Reply:
No that's sadly not possible, OpenAPI only supports the following parameters:

- path parameters, such as `/users/{id}`
- query parameters, such as `/users?role=admin`
- header parameters, such as `X-MyHeader: Value`
- cookie parameters, which are passed in the Cookie header, such as `Cookie: debug=0; csrftoken=BUSe35dohU3O1MZvDCU`

(See https://swagger.io/docs/specification/describing-parameters/)

The "query" in that case corresponds to the URL query component as described in https://datatracker.ietf.org/doc/html/rfc3986#section-3.4 Basically everything after `?` is a query parameter, regardless what it's used for. So it does not mean "query" in the sense of "querying search results".

Stoplight then lists these different kinds of parameters under various titles, for example Headers, Query parameters, etc...

Reply:
Thanks for clarifying this!